Analyze Variants with BigQuery

mbookman opened the pull request:
Rendered version can be viewed at:

http://googlegenomics.readthedocs.org/en/staging-2/use_cases/analyze_variants/analyze_variants_with_bigquery.html

deflaux commented on docs/source/conf.py:
```diff
@@ -116,6 +116,9 @@
 .. _UCSC Sequence and Annotation Data: http://hgdownload.cse.ucsc.edu/
 .. _Tute's documentation: https://docs.google.com/document/d/1_Kryc4qAqw1NRezaqDJ1tXUSCbxEkKK4SSi_kZuyHtU/pub
 
+.. _Illumina Platinum Genomes project data: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_public_data/platinum_genomes.html
```
There is another way to refer to pages within the site.  See the following restructured text at the top of http://googlegenomics.readthedocs.org/en/latest/workshops/bioc-2015.html

_We will work with both the :doc:`/use_cases/discover_public_data/1000_genomes` reads and variants and also the :doc:`/use_cases/discover_public_data/platinum_genomes` gVCF variants._

deflaux commented:
Wow - this is terrific @mbookman - yay!!!  LGTM

One thing perhaps to add now or later is to emphasize _more_ that data in both gVCF and Complete Genomics masterVar will have non-variant segments.  CGI data is mentioned once in here, but it might be work mentioning in the section that talks about non-variant segments for the first time.

mbookman commented:
Good suggestion. I have added a mention of CGI masterVar files to the non-variant segments discussion.
Also added the new page to the Analyze Variants index page.
